admin-core/assets/src/common/components/AdminHeader.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Disclosure } from '@headlessui/react'
+import { Link } from 'react-router-dom';
+import ChangeLogPopup from './ChangeLogPopup';
+import PluginStats from './PluginStats';
+import { uagbClassNames } from '@Utils/Helpers';
+
+const AdminHeader = ( props ) => {
+	const {
+		iconLogo,
+		title,
+		children,
+	} = props;
+
+	return (
+		<Disclosure as='nav' className='bg-white shadow'>
+			<div className='max-w-3xl mx-auto px-6 lg:max-w-7xl'>
+				<div className='relative flex justify-between h-16'>
+					<div className='flex-1 flex items-center justify-center sm:items-stretch sm:justify-start'>
+						{ iconLogo ? (
+							<>
+								<div className='flex-shrink-0 flex items-center'>
+									<img
+										className={ uagbClassNames( [
+											'h-10',
+											title && 'mr-6',
+										] ) }
+										src={ uag_react.logo_icon_url }
```

Comment:
```suggestion
										src={ uag_react?.logo_icon_url }
```
Please verify if we should have these check.
also check for other localized data used.